tests/api_tests/nodes/test_views.py

```diff
@@ -1352,7 +1467,7 @@ def test_filtering_on_invalid_field(self):
         assert_equal(res.status_code, 400)
         errors = res.json['errors']
         assert_equal(len(errors), 1)
-        assert_equal(errors[0]['detail'], 'Query string contains an invalid filter.')
+        assert_equal(errors[0]['detail'], "'invalid' is not a valid field for this endpoint.")
```

Comment:
We are returning the invalid field names back to the client in our error messages. The assumption is that these could be rendered as HTML.